From a26ed5502622c22daa175861fa9d651a32131cf5 Mon Sep 17 00:00:00 2001
From: Marc-Antoine Perennou <Marc-Antoine@Perennou.com>
Date: Sat, 16 Feb 2019 08:47:45 +0100
Subject: [PATCH 1/2] pam-exherbo: drop legacy pam_systemd argument

kill-session-processes=1 is long gone

Signed-off-by: Marc-Antoine Perennou <Marc-Antoine@Perennou.com>
(cherry picked from commit 13e6666b1e1000d757ac5621a83105110f55e3e3)
---
 data/pam-exherbo/gdm-launch-environment.pam | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/pam-exherbo/gdm-launch-environment.pam b/data/pam-exherbo/gdm-launch-environment.pam
index 8c13b3bc..51a8e003 100644
--- a/data/pam-exherbo/gdm-launch-environment.pam
+++ b/data/pam-exherbo/gdm-launch-environment.pam
@@ -8,7 +8,7 @@ auth        required    pam_permit.so
 
 password    required    pam_deny.so
 
--session    optional    pam_systemd.so kill-session-processes=1
+-session    optional    pam_systemd.so
 session     optional    pam_keyinit.so force revoke
 session     required    pam_succeed_if.so audit quiet_success user = gdm
 session     required    pam_permit.so
-- 
2.17.0

